OmsAgent/Fairfax/omsagent_ff.py

```diff
@@ -112,6 +113,10 @@ def main():
             message = 'Install failed with exit code {0}: please install ' \
```

Comment:
We do not have too, error code for extension are documented on GitHub if someone needs to dig further. Also the bundle installation messages are logged in extension log. I can setup a machine for you to show.